Tasks/AzureCLIV1/azureclitask.ts

```diff
@@ -98,11 +98,24 @@ export class azureclitask {
 
     private static loginAzureRM(connectedService: string): void {
         var servicePrincipalId: string = tl.getEndpointAuthorizationParameter(connectedService, "serviceprincipalid", false);
-        var servicePrincipalKey: string = tl.getEndpointAuthorizationParameter(connectedService, "serviceprincipalkey", false);
+        let authType: string = tl.getEndpointAuthorizationParameter(connectedService, 'authenticationType', true);
```

Comment:
are you backfilling the auth-type into all endpoints? Should we handle if this field is not present?

Reply:
if the field is not present, we fall-back to service principal key as default.

Reply:
Got it. Missed the optional: true